bciers/apps/compliance/src/app/utils/getRequestIssuanceTrackStatusData.tsx

```diff
@@ -0,0 +1,52 @@
+export enum IssuanceStatus {
+  APPROVED = "approved",
+  AWAITING = "awaiting",
+}
+
+export interface RequestIssuanceTrackStatusData {
+  operation_name: string;
+  earned_credits: number;
+  issuance_status: string;
+  bccr_trading_name: string;
+  directors_comments: string;
+}
```

Comment:
nit: We can use `bciers/apps/compliance/src/app/types.ts` to keep all compliance related types/interfaces.